controller/src/main.rs

```diff
@@ -54,14 +57,28 @@ async fn main() -> Result<()> {
             futures::future::ready(())
         });
 
-    // TODO if either of these fails, we should write to the k8s termination log and exit.
+    // Setup Http server to vend prometheus metrics
+    let exporter = opentelemetry_prometheus::exporter().init();
+    let prometheus_server = HttpServer::new(move || {
+        App::new()
+            .app_data(Data::new(exporter.clone()))
```

Comment:
nit: I think this `clone()` might be unnecessary?

Reply:
It would get the following error.
```
error[E0507]: cannot move out of `exporter`, a captured variable in an `Fn` closure
  --> controller/src/main.rs:64:33
   |
61 |     let exporter = opentelemetry_prometheus::exporter().init();
   |         -------- captured outer variable
...
64 |             .app_data(Data::new(exporter))
   |                                 ^^^^^^^^ move occurs because `exporter` has type `PrometheusExporter`, which does not implement the `Copy` trait
```